snuba/consumer.py

```diff
@@ -48,9 +48,12 @@ def __init__(
     def process_message(
         self, message: Message[KafkaPayload]
     ) -> Optional[ProcessedMessage]:
-        # TODO: consider moving this inside the processor so we can do a quick
-        # processing of messages we want to filter out without fully parsing the
-        # json.
+        pre_filter = (
+            enforce_table_writer(self.__dataset).get_stream_loader().get_pre_filter()
+        )
```

Comment:
Should this be moved to a private instance variable instead of performing this lookup chain every time?